editor/gui/editor_bottom_panel.cpp

```diff
@@ -247,6 +248,13 @@ EditorBottomPanel::EditorBottomPanel() {
 	button_hbox->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	bottom_hbox->add_child(button_hbox);
 
+#ifdef ALLOW_AUTO_UPDATE
+	EngineUpdateLabel *eul = memnew(EngineUpdateLabel);
+	eul->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
+	eul->enable_compact_mode();
+	bottom_hbox->add_child(eul);
+#endif
```

Comment:
I wonder if we even need it in the editor. I think having it in the project manager might be good enough.
If a user chooses to stay on an older version for one reason or another, but still wants to keep track of the latest, it makes sense to me to see the update message in the PM but not to have it nag you constantly during development in the editor.

We're also constrained in width in the editor bottom panel, this message being constantly there will quickly become a problem on smaller screens.

Reply:
This option is for users who don't use Project Manager. But I guess not using it at all is not that common 🤔

Reply:
Yeah I thought about that too. It's true alternative project managers like `godots` are rising in popularity and likely imply skipping the default project manager. But those alternative PMs also do their own update checking logic, so it might not be much of a problem.

Reply:
What I mean is launching project directly, like directly from the folder or using e.g. VS Code. Personally I open projects with VS Code most of the time.